pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -197,6 +197,9 @@ type MonitoringSpec struct {
 
 	// ExternalMgrEndpoints points to an existing Ceph prometheus exporter endpoint
 	ExternalMgrEndpoints []v1.EndpointAddress `json:"externalMgrEndpoints,omitempty"`
+
+	// ExternalMgrPort Prometheus exporter port
```

Comment:
```suggestion
	// ExternalMgrServicesPort Prometheus exporter port
```

Some linters complain without this.